src/onboarding/Sign.js

```diff
@@ -41,7 +41,7 @@ class SignContent extends React.PureComponent {
       <React.Fragment>
         <Title>
           <Text size="great" weight="bold" color={theme.textDimmed}>
-            Sign transactions
+            Create transactions
```

Comment:
Actually, I think "Sign" in this case makes sense, since we're asking the user to sign transactions that are popping up. It's likely more confusing as "create".

![image](https://user-images.githubusercontent.com/4166642/45443150-2ecf8f80-b6c4-11e8-88d1-0e8f2e24ad8e.png)

Reply:
I tend to prefer “sign” but I guess “create” would work too. If we decide to change for “create”, we should align everything to it, and it’s also something we should document in our design system to make sure we use this term everywhere.

For example in the signing panel:

<img width="448" alt="image" src="https://user-images.githubusercontent.com/36158/45755149-8b2b3580-bc15-11e8-8bc2-8766dcf99877.png">

Reply:
In the case of the signing panel, I agree with the action being "create" because when you click the button, you _create_ a transaction that pops up in metamask (where you then sign it).

However, in the case of onboarding, those transactions are created automatically, requiring no action on your part, and it's up to you to sign them in metamask.

Reply:
For this specific instance, "Sign transactions" makes sense.